### PR TITLE
Add Show Course Staff option and exclude all course roles by default

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** By default, don't show users that have a course role, and add a filter to show users with a course role

JIRA: [EDUCATOR-5571](https://openedx.atlassian.net/browse/EDUCATOR-5571)

**What changed?**

- By default, don't show users that have a course role, and add a filter to show users with a course role
Screen Shot 2021-04-08 at 3 11 43 PM

**Developer Checklist**
- [x] Test suites passing
- [x] Received code-owner approving review
- [x] Bumped version number [package.json](../package.json)

**Testing Instructions**

This requires https://github.com/edx/edx-platform/pull/26994 to be running locally.
Follow the same setup steps in that PR:

```
-Log in to LMS
-Navigate to a course with users enrolled (the managements command create_test_users may be useful here)
-Under the Instructor Dashboard -> Membership -> Course Team Management, add enrolled learners to the following roles(see below*):
+ Staff
+ Admin
+ Beta Tester
+ Course Data Researcher
```

-Go to the gradebook for that course.
-You should not see any users with course roles.
-Check the "Show Course Staff" box.
-You should now see enrolled users with roles.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by jansenk** [comment]
I need to write tests, just wanted to get the general concept out there and reviewed

File: src/components/FilterBadges/index.jsx [hunk 1]
```diff
@@ -113,10 +118,16 @@ function FilterBadges({
       />
       <SingleValueFilterBadge
         displayName="Cohort"
-        filterName="track"
+        filterName="cohort"
```
Comment (jansenk) [inline src/components/FilterBadges/index.jsx:121]: this wasn't anything to do with my change, but uh, should be changed

**Review by nsprenkle — COMMENTED** [review]
Visual review looks good. Will try to verify locally.

**Comment by nsprenkle** [comment]
> Visual review looks good. Will try to verify locally.

Have verified new behavior locally, ✅ when tests are done

**Comment by jansenk** [comment]
The LMS and bulk-grades PRs have been merged, so as long as you have master installed locally for both, behavior should be there as expected

**Review by nsprenkle — APPROVED** [review]
Verified locally :shipit:
